perf-test/components/SkottiePlayer/index.tsx

```diff
@@ -33,26 +36,38 @@ export default function Skottie ({ lottieURL, width, height }: Props) {
     const id = uuidv4();
     setId(id);
 
+    const dpr = window.devicePixelRatio || 1
+    setRatio(dpr);
```

Comment:
`const dpr = window.devicePixelRatio` ?

Reply:
This is common tweak for case `devicePixelRatio` is not supported by browser, returns zero value and someone tries to pollute this value because window object is global object itself, easily modified by external things.

JS dev should take over this exception in usual cases. We shouldn't trust browser always returns valid value with global variable.

But it's not necessary, if we really don't care of this, we can simply go with `window.devicePixelRatio`.

Reply:
> This is common tweak for case `devicePixelRatio` is not supported by browser, r

What if window.devicePixelRatio is 0.75? it will forcely set to 1?

Reply:
This case has a value, adapt `0.75`. returns right value only when the value is null or undefined or 0(zero)